pkg/api/opssight/v1/types.go

```diff
@@ -107,11 +108,15 @@ type PodPerceiver struct {
 
 // Perceiver stores the Perceiver configuration
 type Perceiver struct {
-	EnableImagePerceiver      bool          `json:"enableImagePerceiver"`
-	EnablePodPerceiver        bool          `json:"enablePodPerceiver"`
-	PodPerceiver              *PodPerceiver `json:"podPerceiver,omitempty"`
-	AnnotationIntervalSeconds int           `json:"annotationIntervalSeconds"`
-	DumpIntervalMinutes       int           `json:"dumpIntervalMinutes"`
+	EnableImagePerceiver             bool          `json:"enableImagePerceiver"`
+	EnableArtifactoryPerceiver       bool          `json:"enableArtifactoryPerceiver"`
+	EnableArtifactoryPerceiverDumper bool          `json:"enableArtifactoryPerceiverDumper"`
```

Comment:
why 2 enables Artifactory Perceiver?

Reply:
The second enables dumping all artifactory images into perceptor. Normally it'd work with webhooks, if something is pushed it adds to perceptor.